astropy/io/fits/convenience.py

```diff
@@ -443,7 +444,7 @@ def writeto(filename, data, header=None, output_verify='exception',
 
 def table_to_hdu(table):
     """
-    Convert an astropy.table.Table object to a FITS BinTableHDU
+    Convert an `~astropy.table.Table` object to a FITS BinTableHDU
```

Comment:
Oh man, I don't know if I want to go down the rabbit hole of fixing other people's doc strings  in this file, but I can at least change this one since you mentioned it.